js/viz/chart.js

```diff
@@ -880,6 +888,46 @@ const dxChart = AdvancedChart.inherit({
         return rangeDataCalculator.getViewPortFilter(this.getArgumentAxis().visualRange() || {});
     },
 
+    _applyAutoHidePointMarkers(filteredSeries) {
+        let overlapPoints = [];
+        filteredSeries.reduceRight((_, currentSeries) => {
+            currentSeries.autoHidePointMarkers = false;
+
+            if(currentSeries.autoHidePointMarkersEnabled()) {
+                let counterOverlapSingle = 0;
+                const seriesPoints = currentSeries._points;
+
+
+                seriesPoints.forEach((currentPoint, index) => {
+                    if(cycleComparison(index + 1, seriesPoints, currentPoint, false)) {
+                        counterOverlapSingle++;
+                    }
+                    if(counterOverlapSingle > (seriesPoints.length / 2)) {
```

Comment:
Maybe this 'If' move out from cycle 'forEach'?

Reply:
Fixed